rm/rm-policy/rm-policy-scheduler/src/main/java/org/ow2/proactive/scheduler/resourcemanager/nodesource/policy/SchedulerAwarePolicy.java

```diff
@@ -88,7 +89,11 @@ public BooleanWrapper configure(Object... policyParameters) {
         schedulerUrl = policyParameters[2].toString();
 
         if (policyParameters[3] == null) {
-            throw new IllegalArgumentException("Credentials must be specified");
+            try {
+                policyParameters[3] = nodeSource.getAdministrator().getCredentials().getBase64();
+            } catch (KeyException e) {
+                logger.info("Cannot use default admin credentials.", e);
+            }
```

Comment:
If instead throwing an exception when `policyParameters[3] == null` you log the error. I think it will return a NullPointerException when you do `credentials = (byte[]) policyParameters[3]`.

Reply:
thanks. i changed